x/gov/module.go

```diff
@@ -160,6 +160,7 @@ func init() {
 		appmodule.Invoke(InvokeAddRoutes, InvokeSetHooks))
 }
 
+//nolint:revive
```

Comment:
I see very much nolint of revive. Can we not use it instead?

Reply:
I'd prefer to use it, and migrate towards its naming conventions because that gives another point of standardization.  In this case it would tell us that the call stutters, because gov.GovInputs

Reply:
Alright 👍🏾